BUILD.md

```diff
@@ -0,0 +1,173 @@
+# Building the Arkouda Server
+
+## Table of Contents
+1. [Getting Started](#start)
+2. [Environment Variables](#env-vars)
+3. [Building the Source](#build-ak-source)
+   1. [Using Environment Installed Dependencies](#env_deps)
+   2. [Installing Dependencies](#install-deps)
+
+<a id="start"></a>
+## Getting Started <sup><sup><sub><a href="#toc">toc</a></sub></sup></sup>
+ 
+If you have not installed the Arkouda Client and prerequisites, please follow the directions in [INSTALL.md](INSTALL.md) before proceeding with this build.
+
+Download, clone, or fork the [arkouda repo](https://github.com/mhmerrill/arkouda). Further instructions assume that the current directory is the top-level directory of the repo.
+
+<a id="env-vars"></a>
+## Environment Variables <sup><sup><sub><a href="#toc">toc</a></sub></sup></sup>
+In order to build the server executable, some environment variables need to be configured. For a full list, please refer to [ENVIRONMENT.md](ENVIRONMENT.md).
+
+<a id="build-ak-source"></a>
+## Build the Source <sup><sup><sub><a href="#toc">toc</a></sub></sup></sup>
+
+<a id="env-deps"></a>
+### Using Environment Installed Dependencies <sup><sup><sub><a href="#toc">toc</a></sub></sup></sup>
+If your environment requires non-system paths to find dependencies (e.g., if using the ZMQ and HDF5 bundled with [Anaconda]), append each path to a new file `Makefile.paths` like so:
```

Comment:
I'm having trouble parsing this line. I feel like the `Makefile.paths` should only contain one line in this situation which is the path to your env, right?

Reply:
This is what we had before. In most cases, yes we only need to provide a single line. However, if someone manually installs things, they could have multiple. I will retool the wording some, but we should not be stating this should only have 1 line because it may not.